### PR TITLE
Fix isssues #178

### LINKED ISSUE #178: Create class Minutes
```
$minutes = new Minutes(0);
$minutes->getStart() // return null
```
line 24 in Minutes.php

```
if ( ! $minutes) // !0 return true
        {
            return;
        }
```

### PR DESCRIPTION
**Pull request by aleplusplus**
Closes #178

### CODE REVIEW

**Comment by aleplusplus** [comment]
Fix issues #133 and #142

**Comment by antonioribeiro** [comment]
If you touch code you're are not really changing, and you touched a lot, it make things very hard to merge.

**Comment by antonioribeiro** [comment]
Once lots and lots of people are using a package, you don't change migrations, you create a new ones.

File: src/migrations/2015_11_23_311097_create_tracker_language_table.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+use PragmaRX\Tracker\Support\Migration;
+
+class CreateTrackerLanguagesTable extends Migration {
```
Comment (antonioribeiro) [inline src/migrations/2015_11_23_311097_create_tracker_language_table.php]: Class is incorrectly named. Should be CreateTrackerLanguageTable.
But I fixed in my end.

**Comment by antonioribeiro** [comment]
Thank you!

**Pull request merged**